ompi/runtime/mpiruntime.h

```diff
@@ -55,6 +55,8 @@ OMPI_DECLSPEC extern bool ompi_mpi_initialized;
 OMPI_DECLSPEC extern bool ompi_mpi_finalized;
 /** Has the RTE been initialized? */
 OMPI_DECLSPEC extern bool ompi_rte_initialized;
+/** Did mpi start to finalize? */
+OMPI_DECLSPEC extern int32_t ompi_mpi_finalize_started;
```

Comment:
It's probably 6 of one, half dozen of the other.  I minorly prefer keeping all the types together on all branches.  It does seem a little weird to commit something that we know is not what we want in the final state.  Is there much additional QA burden to change from int32_t to bool?